dropshot/Cargo.toml

```diff
@@ -54,4 +54,6 @@ features = [ "uuid" ]
 
 [dev-dependencies]
 libc = "0.2.71"
-difference = "2.0.0"
+difference = "2.0"
```

Comment:
Nit: I think in a previous PR @steveklabnik mentioned that "x.y.0" was the same as "x.y", which I think is because [Cargo implicitly uses the "^" syntax](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio).  I thought we decided to be consistent and always specify a patch version, even if it's zero.  WDYT?

Reply:
yes, these two lines are equivalent. I personally always put all three numbers, but the tools will support either of these until the end of all time

Reply:
did not know! thank you.